src/Components/People.js

```diff
@@ -0,0 +1,75 @@
+import { Component } from "react";
+import axios from "axios";
+
+class People extends Component {
+  state = { input: "", people: [], hasSearched: false, ids: {} };
+
+  handleSubmit = async (e) => {
+    e.preventDefault();
+    this.setState({ hasSearched: true });
+    try {
+      const { input } = this.state;
+      const res = await axios.get(`https://ghibliapi.herokuapp.com/people/`);
+      this.setState({ people: res.data });
+      
+    } catch (err) {
+      this.setState({ people: {} });
+    }
+    this.setState({ input: "" });
+    debugger;
+    {
+      const names = this.state.people.map((movie) => {
```

Comment:
I think right here is a great place for a filter or find method. You can use these to access the one person whose name matches the user's input. Once you have this you can set it in state and use it to display their information